<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ground Station</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="main_tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mapTab">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="DataDisplayWidget" name="mapWidgetPlaceholder"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="graphTab">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="altitudeGraphDisplayWidget" name="altitudeGraphWidget">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="velocityGraphDisplayWidget" name="velocityGraphWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="controlsTab">
       <attribute name="title">
        <string>Controls</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="DecoderWidget" name="controlsWidgetPlaceholder" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DecoderWidget</class>
   <extends>QWidget</extends>
   <header>data_sources/dummy_source.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DataDisplayWidget</class>
   <extends>QGraphicsView</extends>
   <header>dummy_map</header>
  </customwidget>
  <customwidget>
   <class>altitudeGraphDisplayWidget</class>
   <extends>QGraphicsView</extends>
   <header>graphs.h</header>
  </customwidget>
  <customwidget>
   <class>velocityGraphDisplayWidget</class>
   <extends>QGraphicsView</extends>
   <header>graphs.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
